tests/Exception/RequestExceptionTest.php

```diff
@@ -60,18 +60,27 @@ public function testCreatesServerErrorResponseException()
 
     public function testCreatesGenericErrorResponseException()
     {
-        $e = RequestException::create(new Request('GET', '/'), new Response(600));
+        $e = RequestException::create(new Request('GET', '/'), new Response(300));
         $this->assertContains(
             'GET /',
             $e->getMessage()
         );
         $this->assertContains(
-            '600 ',
+            '300 ',
             $e->getMessage()
         );
         $this->assertInstanceOf('GuzzleHttp\Exception\RequestException', $e);
     }
 
+    /**
+     * @expectedException InvalidArgumentException
+     * @expectedExceptionMessage Status code must be an integer value between 1xx and 5xx.
+     */
+    public function testThrowsInvalidArgumentExceptionOnOutOfBoundsResponseCode()
```

Comment:
This should have been removed.. since the exception comes from PSR-7 package and this is already tested there...